packages/theme-chalk/src/select-v2.scss

```diff
@@ -217,6 +217,10 @@ $input-inline-start: map.get($input-padding-horizontal, 'default') !default;
       #{getCssVarName('input-icon-color')},
       map.get($input, 'icon-color')
     );
+
+    & .#{$namespace}-input__icon {
+      height: inherit;
+    }
```

Comment:
**Suggestion:** remove `&` here since it behaves the same w/ or w/o this symbol.

Reply:
OK.